### PR TITLE
Fall back to ANDROID_NDK if NDK not set

### PR DESCRIPTION
**Pull request by marmistrz**

### CODE REVIEW

File: kodev [hunk 1]
```diff
@@ -94,6 +94,10 @@ ${SUPPORTED_TARGETS}"
             assert_ret_zero $?
             ;;
         android)
+            if [ "$NDK" == "" ]; then
+                # Some distributions use `ANDROID_NDK` instead, fall back to it
```
Comment (Frenzie) [inline kodev]: How about something like this instead?

```bash
if [ -z "$NDK" ]; then
	if [ -z "$ANDROID_NDK" ]; then
		# some distributions use `ANDROID_NDK` instead, fall back to it
		export NDK=$ANDROID_NDK
	else
		export NDK="${CURDIR}/base/toolchain/android-ndk-r12b"
	fi
fi
```

Comment (marmistrz) [inline kodev]: You mean `if [ ! -z "$ANDROID_NDK" ]; then` ?

Comment (marmistrz) [inline kodev]: You mean
```
if [ -z "$NDK" ]; then
	if [ -z "$ANDROID_NDK" ]; then
		export NDK="${CURDIR}/base/toolchain/android-ndk-r12b"
	else
		# some distributions use `ANDROID_NDK` instead, fall back to it
		export NDK=$ANDROID_NDK
	fi
fi
```

Comment (Frenzie) [inline kodev]: I meant `[ ${ANDROID_NDK+x} ]` (no `-z`, sorry for the typo) but it doesn't matter either way. It just feels inelegant to do the same check twice.

Comment (marmistrz) [inline kodev]: So,
```bash
if [ ${NDK+x} ]; then
	if [ ${ANDROID_NDK+x} ]; then
		# some distributions use `ANDROID_NDK` instead, fall back to it
		export NDK=$ANDROID_NDK
	else
		export NDK="${CURDIR}/base/toolchain/android-ndk-r12b"
	fi
fi
```

Comment (Frenzie) [inline kodev]: Yes, it should.

* `-z ${NDK+x}` (unset or empty)
* `-n ${ANDROID_NDK+x}` (set and non-empty)

**Comment by Frenzie** [comment]
Thanks! I hope you don't mind I went ahead and merged the discussed changes. :-)

**Comment by marmistrz** [comment]
Yeah, that's ok, I was a little busy back then :)

**Pull request merged**